File: src/Qwack.Transport/TransportObjects/MarketData/VolSurfaces/TO_VolSurface.cs
using ProtoBuf;

namespace Qwack.Transport.TransportObjects.MarketData.VolSurfaces
{
    [ProtoContract]
    public class TO_VolSurface
    {
        [ProtoMember(4)]
        public TO_ConstantVolSurface ConstantVolSurface { get; set; }
        [ProtoMember(5)]
        public TO_GridVolSurface GridVolSurface { get; set; }
        [ProtoMember(6)]
        public TO_RiskyFlySurface RiskyFlySurface { get; set; }
        [ProtoMember(7)]
        public TO_SparsePointSurface SparsePointSurface { get; set; }
    }
}

File: src/Qwack.Transport/GeneratedTypeScript/TO_VolSurface.ts

import { TO_ConstantVolSurface } from './TO_ConstantVolSurface';
import { TO_GridVolSurface } from './TO_GridVolSurface';
import { TO_RiskyFlySurface } from './TO_RiskyFlySurface';

export interface TO_VolSurface  {
	 constantVolSurface: TO_ConstantVolSurface;
	 gridVolSurface: TO_GridVolSurface;
	 riskyFlySurface: TO_RiskyFlySurface;
}


